src/DurableTask.Netherite/OrchestrationService/OrchestrationMessageBatch.cs

```diff
@@ -162,20 +169,36 @@ public void ConfirmDurable(Event evt)
             this.workItem.Partition.EnqueueOrchestrationWorkItem(this.workItem);
         }
 
-        bool IsExecutableInstance(TaskOrchestrationWorkItem workItem, out string message)
+        bool IsExecutableInstance(TaskOrchestrationWorkItem workItem, out string message, out bool isNormalSituationForTimerFired)
         {
+            message = default;
+
             if (workItem.OrchestrationRuntimeState.ExecutionStartedEvent == null
                 && !this.MessagesToProcess.Any(msg => msg.Event is ExecutionStartedEvent))
             {
                 if (DurableTask.Core.Common.Entities.AutoStart(this.InstanceId, this.MessagesToProcess))
                 {
-                    message = default;
+                    isNormalSituationForTimerFired = default;
```

Comment:
nit: for booleans, I'd explicitly specify the default. I think "default" is best used for values that don't directly affect control-flow like booleans. That said, I realize in this case it's pretty obvious the default is `false` so I'll leave this change up to you.